### PR TITLE
update the url of github projects that have moved

### PR DESCRIPTION
**Pull request by robotvert**
Accessing them using a browser makes github redirect you to the new home but using the github API these throw 404s, which is annoying to deal with.

I don't know the policy around old URLs, maybe such a check should be part of the guidelines and the test suite.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ Join us on IRC at **#awesome-go** on freenode [web access](http://webchat.freeno
 * [go-log](https://github.com/siddontang/go-log) - Log lib supports level and multi handlers.
 * [go-log](https://github.com/ian-kent/go-log) - A log4j implementation in Go.
 * [go-logger](https://github.com/apsdehal/go-logger) - Simple logger of Go Programs, with level handlers.
-* [gologex](https://github.com/chzyer/gologex) - An golang log lib, supports tracking and level, wrap by standard log lib
+* [gologex](https://github.com/go-logex/logex) - An golang log lib, supports tracking and level, wrap by standard log lib
```
Comment (shurcooL) [inline README.md]: The package name has changed to `logex`, so it should be renamed and re-ordered.

File: README.md [hunk 2]
```diff
@@ -452,7 +452,7 @@ Join us on IRC at **#awesome-go** on freenode [web access](http://webchat.freeno
 *Libraries for using OpenGL in Go.*
 
 * [gl](https://github.com/go-gl/gl) - Go bindings for OpenGL. Requires an external dependency GLEW.
-* [glfw3](https://github.com/go-gl/glfw3) - Go bindings for GLFW 3.
+* [glfw3](https://github.com/go-gl/glfw) - Go bindings for GLFW 3.
```
Comment (shurcooL) [inline README.md]: Thanks for catching this. It should also be renamed (and re-ordered) to `glfw` now.

File: README.md [hunk 3]
```diff
@@ -566,7 +566,7 @@ Join us on IRC at **#awesome-go** on freenode [web access](http://webchat.freeno
     * [counterfeiter](https://github.com/maxbrunsfeld/counterfeiter) - Tool for generating self-contained mock objects
     * [go-sqlmock](https://github.com/DATA-DOG/go-sqlmock) - Mock SQL driver for testing database interactions
     * [gomock](https://code.google.com/p/gomock/) - Mocking framework for the Go programming language.
-    * [mockhttp.go](https://github.com/tv42/mockhttp.go) - Mock object for Go http.ResponseWriter
+    * [mockhttp.go](https://github.com/tv42/mockhttp) - Mock object for Go http.ResponseWriter
```
Comment (shurcooL) [inline README.md]: This is renamed to `mockhttp`.

**Comment by shurcooL** [comment]
Overall, I checked the redirects, and everything is correct. Thank you for fixing them!

Could you address my comments about renamed packages above? After that, this should be good to merge.

I'll make a followup PR to fix up the OpenGL section, as it's out of date.

**Comment by robotvert** [comment]
Sorry about this, I totally skipped the renaming & re-sorting part.

BTW at around line 193 the markdown seems broken as the line 
> Libraries for connecting and operating databases.

doesn't have a closing `*`.

I didn't address this for the sake of having a single concern PR.

**Comment by shurcooL** [comment]
Thank you!

LGTM.

**Pull request merged**